core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -236,4 +243,27 @@ private static void deleteFiles(FileIO io, Set<ManifestFile> allManifests) {
           }
         });
   }
+
+  private static String fullTableName(String catalogName, TableIdentifier identifier) {
+    StringBuilder sb = new StringBuilder();
+
+    if (catalogName.contains("/") || catalogName.contains(":")) {
```

Comment:
Since now `catalogName` doesn't contain "/" or ":", do we still need such code?

Reply:
Yes, I think we do. Other catalogs that extend `BaseMetastoreCatalog` may choose to use a URI for catalog name like the original PR. I think that's a reasonable format.